.gitattributes

```diff
@@ -0,0 +1,22 @@
+# Set default behaviour, in case users don't have core.autocrlf set.
+*       text=auto
+
+# Explicitly declare text files we want to always be normalized and converted 
+# to native line endings on checkout.
+*.java  text
+*.xml   text
+*.xsl   text
+*.txt   text
+
+# Test files that should be left alone.
+*.csv   -text
```

Comment:
This entry isn't specific to this repo and can be removed.

Reply:
I removed entries that doidn't seem likely to be used in USNG.js.